core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/CarbonUtils.java

```diff
@@ -80,6 +92,9 @@
             "org.wso2.carbon.tomcat.ext.transport.ServletTransportManager";
 	private static final String TRUE = "true";
 	private static Log log = LogFactory.getLog(CarbonUtils.class);
+    private static final int ENTITY_EXPANSION_LIMIT = 0;
+    private static final String SECURITY_MANAGER_PROPERTY = org.apache.xerces.impl.Constants.XERCES_PROPERTY_PREFIX +
```

Comment:
Entity Expansion Limit, if used in different classes, can be different from place to place. But at the moment this is the only place it is used. Should we move it as public?
We can make SECURITY_MANAGER_PROPERTY public and can move it to constant class.